lib/session.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
+
+/**
+ * Session model.
+ */
+
+class Session {
+  /**
+   * Session constructor
+   * @param {Context} ctx
+   * @param {Object} obj
+   * @api private
+   */
+
+  constructor(ctx, obj) {
+    this._ctx = ctx;
```

Comment:
ContextSession named `ctx` but here is `_ctx`, what's the different on naming ctx property?